components/ide/jetbrains/backend-plugin/src/main/resources-stable/META-INF/extensions.xml

```diff
@@ -6,5 +6,7 @@
 <idea-plugin>
     <extensions defaultExtensionNs="com.intellij">
         <projectService serviceImplementation="io.gitpod.jetbrains.remote.stable.GitpodTerminalService" preload="true"/>
+        <gateway.customization.performance id="gitpodMetricsControl" order="before cpuControl" implementation="io.gitpod.jetbrains.remote.latest.GitpodMetricControlProvider"/>
```

Comment:
Should not we support them in latest as well?

Reply:
common part should go into https://github.com/gitpod-io/gitpod/blob/6ce6daa95e8fbefe87024dc67d2b9273cb32d467/components/ide/jetbrains/backend-plugin/src/main/resources/META-INF/plugin.xml#L24

Reply:
Thanks for the info, fixed